docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -129,11 +127,7 @@ In the event of a total active region loss, the following data will be lost:
   - Role Based Access Control (RBAC) does not work.
 - Optimize is not supported.
   - This is due to Optimize depending on Identity to work.
-- Connectors are not supported.
-  - This is due to Connectors depending on Operate to work for inbound Connectors and potentially resulting in race condition.
-- During the failback procedure, there’s a small chance that some data will be lost in Elasticsearch affecting Operate and Tasklist.
-  - This **does not** affect the processing of process instances in any way. The impact is that some information about the affected instances might not be visible in Operate and Tasklist.
-  - This is further explained in the [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md?failback=step2#failback) during the relevant step.
+- Connectors can be deployed alongside but ensure to understand idempotency based on [the described documentation](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event).
```

Comment:
Could this be more declarative regarding support?

```suggestion
- Connectors are supported, but may require special considerations regarding [idempotency](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event).
```
It's not clear to me from the linked documentation why idempotency would be more of a concern in this setup, but that may be my lack of familiarity. Is it _always_ a concern in this setup? Should they always use the message start event element? Some clarity here would be great.

Reply:
I don't want to use the word "supported" atm since we have an extra ticket to actually create tests / manifest adjustments etc. to officially support it.
It's just that it isn't a limitation anymore but can be done under premise to understand message idempotency.

I've added a subsentence on why it's important.
TL;DR - multiple connectors instances + message idempotency = no duplicated events.
Without it = may end up triggering things multiple times.
Same applies to a single region setup, just that in multi-region by default we have two connector instances.

Reply:
This works for me then. 👍 The additional messaging helps!